instagram/src/components/PostContainer/PostCard.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function PostCard(props) {
+  return (
+    <div className="postCardContainer"></div>
```

Comment:
For CSS class names, you should keep them to be more in line with what you expect from CSS (i.e. `post-card-container`).